packages/design-system-react-native/jest.config.js

```diff
@@ -31,7 +31,7 @@ module.exports = merge(baseConfig, {
     '!**/*.dev.{js,ts}', // Exclude .dev files
     '!**/*.assets.{js,ts}', // Exclude .assets files
     '!**/*.types.{js,ts}', // Exclude .types files
-    '!./src/components/temp-components/Blockies/Blockies.utilities.d.ts',
+    '!./src/components/temp-components/Blockies/Blockies.utilities.ts',
```

Comment:
We don't quite have 100% testing coverage for the Blockies utilities yet so ignoring it for now. I've created an issue to improve the testing coverage here https://github.com/MetaMask/metamask-design-system/issues/700

<img width="1512" alt="Screenshot 2025-05-25 at 2 57 41 PM" src="https://github.com/user-attachments/assets/dd2cac1b-31f2-4234-970e-f93f5aa95159" />